upcoming_changes/2975.doc.rst

```diff
@@ -0,0 +1 @@
+Add a note in the user guide to explain that when a file contains several datasets, :py:func:`~.io.load` returns a list of signals instead of a single signal and that list indexation can be used to access a single signal.
```

Comment:
Shall this changelog note be moved to `RosettaSciIO`?

Reply:
But that part of the documentation was not moved, as far as I understand it. It is in the getting started section of the hypserspy userguide. One could modify this changelog to say that it concerns `getting_started`.

Reply:
Yes, sorry, it was not clear and I have put this comment in the wrong place! I was confused with the changelog entry for #2896, which I have noticed when checking the doc build on readthedocs. Actually, this is not related to this PR because it is already on the `RELEASE_next_major` branch. Sorry for the noise!